tests/register/register.go

```diff
@@ -31,6 +39,33 @@ func init() {
 	Tests = make(map[TestType][]types.Test)
 }
 
-func Register(tType TestType, t types.Test) {
+func register(tType TestType, t types.Test) {
 	Tests[tType] = append(Tests[tType], t)
 }
+
+// Registers t for every version, inside the same major version, that is equal to or greater than the specified ConfigMinVersion.
+func Register(tType TestType, t types.Test) {
+	// update confgiVersions with new config versions
+	configVersions := [][]semver.Version{
+		{semver.Version{}}, // place holder
```

Comment:
What is this a place holder for? Just so that the index lines up to the Major Version?

Reply:
Yes. Should I remove it?

Reply:
I'm not worried about it, just wanted to make sure.